.webpack/webpack.base.js

```diff
@@ -81,8 +81,16 @@ module.exports = (env, argv, { SRC_DIR, DIST_DIR }) => {
         'process.env.BUILD_NUM': JSON.stringify(BUILD_NUM),
         /* i18n */
         'process.env.USE_LOCIZE': JSON.stringify(process.env.USE_LOCIZE || ''),
-        'process.env.LOCIZE_PROJECTID': JSON.stringify(process.env.LOCIZE_PROJECTID || ''),
-        'process.env.LOCIZE_API_KEY': JSON.stringify(process.env.LOCIZE_API_KEY || ''),
+        'process.env.LOCIZE_PROJECTID': JSON.stringify(
+          process.env.LOCIZE_PROJECTID || ''
+        ),
+        'process.env.LOCIZE_API_KEY': JSON.stringify(
+          process.env.LOCIZE_API_KEY || ''
+        ),
+        /* Cadia */
+        'process.env.ANNOTATION_TYPE': JSON.stringify(
+          process.env.ANNOTATION_TYPE || 'MAMA'
+        ),
```

Comment:
Would this be per instance of the OHIF Viewer, or would it make more sense to have it be contextual on a specific workflow, type of study, etc?